addons/mail/wizard/mail_template_preview.py

```diff
@@ -82,6 +84,9 @@ def _compute_mail_template_fields(self):
                 self._set_mail_attributes()
                 error_msg = user_error.args[0]
         self.error_msg = error_msg
+        self.has_several_languages_installed = len(self.env['res.lang'].get_installed()) > 1
```

Comment:
The `lang` field definition is already based on the result of `self.env['res.lang'].get_installed()` through `_selection_languages`, can't we take advantage of that for the length, instead of re-fetching all languages?